wdl-lint/tests/lints/blanks-between-elements/source.wdl

```diff
@@ -0,0 +1,92 @@
+#@ except: DescriptionMissing, ImportWhitespace, InputSorting, LineWidth, MissingMetas, MissingOutput, MissingRuntime, Whitespace
+
+version 1.1
+
+import "baz" # following whitespace will be caught by ImportWhitespace rule
+
+import "qux" # following whitespace duplication is caught be Whitespace rule
+
+
+# test comment
+workflow foo {
+
+    # This is OK.(but the prior line is not)
+    # So is this.
+    meta {}
+    parameter_meta {}
+    # what about this comment?
+    input {}
+    scatter (i in ["hello", "world"]) {
+        call bar { input: s = i }
+    }
+    if (true) {
```

Comment:
I don't see a lint for this one—should there be?

Reply:
No, `scatter`, `if`, `call`, and private declarations do not need blank lines when used in the workflow body. The first instance of one of those after a "section" (input, output, meta, parameter_meta, runtime, hints, requirements) does.